concepts/delta_query_overview.md

```diff
@@ -74,13 +79,9 @@ Delta query is currently supported in preview on the Microsoft Graph /beta endpo
 
 The same [permissions](./permissions_reference.md) that are required to read a specific resource are also required to perform delta query on that resource.
 
-## Known limitations
-
-For known limitations using delta query, see the [delta query section](../overview/release_notes.md#delta-query) in the known issues article.
-
 ## Delta query request examples 
 
-- [Get incremental changes to events in a calendar view (preview)](../Concepts/delta_query_events.md)
-- [Get incremental changes to messages in a folder (preview)](./delta_query_messages.md)
-- [Get incremental changes to groups (preview)](./delta_query_groups.md)
-- [Get incremental changes to users (preview)](./delta_query_users.md)
\ No newline at end of file
+- [Get incremental changes to events in a calendar view](../Concepts/delta_query_events.md)
+- [Get incremental changes to messages in a folder](./delta_query_messages.md)
+- [Get incremental changes to groups](./delta_query_groups.md)
+- [Get incremental changes to users](./delta_query_users.md)
```

Comment:
Would be good to add a link to the existing OneDrive example here: https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/item_delta